src/commonMain/kotlin/software/momento/kotlin/sdk/internal/utils/ValidationUtils.kt

```diff
@@ -28,4 +21,22 @@ public object ValidationUtils {
     internal fun requireValidTtl(ttl: Duration) {
         require(ttl.isPositive()) { CACHE_ITEM_TTL_MUST_BE_POSITIVE }
     }
+
+    internal fun requireValidTruncateToSize(truncateToSize: Int?) {
+        require(truncateToSize == null || truncateToSize > 0) { TRUNCATE_TO_SIZE_MUST_BE_POSITIVE }
+    }
+
+    internal fun requireValidListName(listName: String?) {
+        require(!listName.isNullOrBlank()) { LIST_NAME_IS_REQUIRED }
```

Comment:
This is minor, but are empty list names valid? For some reason I thought the server accepted them.

Reply:
You are right! Empty list names are indeed accepted.